projects/saturn/src/test/resources/test-views.yaml

```diff
@@ -0,0 +1,49 @@
+views:
+  - name: Resource
+    title: Files
+    itemName: Name
+    types:
+      - https://sils.uva.nl/ontology#ExternalFile
+    columns:
+      - name: externalFileLink
+        title: File link
+        source: https://sils.uva.nl/ontology#externalFileLink
+        type: link
```

Comment:
this is metatree specific column type, not supported in Fairspace (that is why the tests are failing). Could you remove the external file link from this config?